tests/unit/Framework/ExecutionOrderDependencyTest.php

```diff
@@ -112,4 +112,11 @@ public function testMergeHandlesEmptyDependencyLists(): void
             'Right side of merge could be empty',
         );
     }
+
+    public function testEmptyClassOrCallable(): void
+    {
+        $empty = new ExecutionOrderDependency('');
```

Comment:
this test failed before the fix in https://github.com/sebastianbergmann/phpunit/pull/5777#discussion_r1544727534